class_materials/Intro2Pandas/2020/tests.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+
+def test_q1_shape(df):
+    try:
+        assert df.shape[0] == 45
+    except AssertionError:
+        print("Your dataframe doesn't have the correct number of rows.")
+        print("Try looking back at conditional statements using loc/iloc.")
+        return
+
+    print("You successfully used pandas to filter a dataset")
+
+
+def test_q2_shape(df):
+    try:
+        assert df.shape[0] == 42
+    except AssertionError:
+        print("Your dataframe doesn't have the correct number of rows.")
+        print("Try looking back at conditional statements using loc/iloc.")
+        return
+
+    print("You successfully filtered the dataset")
+
+
+def test_q2_statement(str):
+    try:
+        assert (
+            str
```

Comment:
```suggestion
def test_q2_statement(statement):
    try:
        assert (
            statement
```
`str` is a python reserved word, so it's safer to use `statement` instead